CODE_REVIEW_CHECKLIST.md

```diff
@@ -69,3 +71,10 @@ may have.
 
 - [ ] Will this change work on all of our supported platforms and
       browsers?
+
+- [ ] Do we expect significant changes in performance due to the
+      changed code? Do we have any indication of the impact related to
+      those performance changes?
+
+- [ ] Should this change be reflected in the release notes? If so, has
```

Comment:
Tried clarifying... But I've also noticed that it's not something external contributors would ever be able to check...